client_connection/interface.go

```diff
@@ -0,0 +1,31 @@
+package client_connection
+
+import (
+	"errors"
+	"github.com/mysterium/node/identity"
+)
+
+type State string
+
+const (
+	NOT_CONNECTED = State("NOT_CONNECTED")
+	NEGOTIATING   = State("NEGOTIATING")
+	CONNECTED     = State("CONNECTED")
```

Comment:
Why are we using ALL_CAPS_CASE for constants as in other programming languages? It should be `NotConnected`, `Negotiating`, `Connected` in Go.

Reply:
I agree, but this convention is throughout the project:)

Reply:
Well we can agree to start using go way constants and this is place would be an example :) Will fix names